### ISSUE TITLE
Dead link to experimental model

### ISSUE THREAD

**Issue by davidpagnon** [CMU-Perceptual-Computing-Lab/openpose_train#61]
Dear OpenPose team,

I use the OpenPose [experimental models](https://github.com/CMU-Perceptual-Computing-Lab/openpose_train/tree/master/experimental_models#body_25b-model---option-2-recommended) for my [Pose2Sim](https://github.com/perfanalytics/pose2sim) toolbox, and it seems like the links to caffe models are dead. 
Example [with this link](http://posefs1.perception.cs.cmu.edu/OpenPose/models/pose/100_135AlmostSameBatchAllGPUs/body_135/pose_iter_XXXXXX.caffemodel). 

Actually, the whole https://posefs1.perception.cs.cmu.edu/ website is inaccessible.

Would you be able to put it back online?

**Comment by JohannesTimter** [comment]
Yes please!

**Comment by GiuseppeLongo25** [comment]
Same problem for me. Link is not avaible. On github there are several models but MODEL 25B it's not online. Any suggestions?

**Comment by Blackwoj** [comment]
I found some models here:
https://github.com/CMU-Perceptual-Computing-Lab/openpose/issues/1602
On google drive, I don't know if it's the same model, but there is something called body_25

**Comment by GiuseppeLongo25** [comment]
> I found some models here: [CMU-Perceptual-Computing-Lab/openpose#1602](https://github.com/CMU-Perceptual-Computing-Lab/openpose/issues/1602) On google drive, I don't know if it's the same model, but there is something called body_25

Yes in that conversation there's something but some models does not work. Now I'm working with only 3 models

**Comment by davidpagnon** [comment]
What do you mean by "they don't work"? What is not working?
And which ones are not working?

**Comment by GiuseppeLongo25** [comment]
On google drive there are several models. Also following folder instrunction in which insert models some of them does not run. When you write the openpose command with the specified model it does not run.
The only model that works are the following one:
- pose_iter_116000.caffemodel that you have to insert in a folder called **face**;
-  pose_iter_102000.caffemodel and pose_iter_120000.caffemodel that you have to insert in **hand** folder;
-  pose_iter_584000.caffemodel in model_25 folder in **pose** folder;
-  pose_iter_440000.caffemodel in coco folder in **pose** folder;
-  pose_iter_160000.caffemodel in mpi folder in **pose** folder.
Every folder is to insert in a **model** folder in **openpose** one.
Other models does not run and it's not specified folder name. Maybe models are okay but i don't know the correct folder name.

**Comment by davidpagnon** [comment]
Did the experimental models use to work with you? I should try it and keep my users informed if it does not work, but I won't have time for it this week.

**Comment by GiuseppeLongo25** [comment]
> Did the experimental models use to work with you? I should try it and keep my users informed if it does not work, but I won't have time for it this week.

Yes. I'm using model 25 and it's working properly.

**Comment by seekFire** [comment]
So how to obtain models in **experimental_models**? Only model_25 can be available?

**Comment by davidpagnon** [comment]
@seekFire I can send them to you if you want, drop me an email at dp2032@bath.ac.uk.
Be aware though, that OpenPose is getting outdated and you'd have better performances with MMPose or RTMlib.

**Comment by seekFire** [comment]
@davidpagnon Thank you very much!
Yes, I know the performance of Openpose is no longer advanced, so I wonder whether all openpose model are bottom-up? Including wholebody135 and body+foot25B?

**Comment by davidpagnon** [comment]
If I remember well, yes they are all bottom-up.
I just sent you a link, can you tell me if you got it?

**Comment by seekFire** [comment]
@davidpagnon Yes, the link is OK! Thank you very much!
Because my job focuses on deployment rather than training, I usually find the trained weight file directly. If all models in OpenPose are bottom-up, it has unique worth for me, especially wholebody135 and body+foot25B. You know the output of most pose model (either top-down or bottom-up) is coco format(only 17 points). I have found some wholebody pose model(for example based on COCO-WholeBody, 135 points), all of them are top-down, you know the inference speed of these models depend on the number of people in the scene. So a bottom-up whole-body model is valuable.

**Comment by davidpagnon** [comment]
Do you need bottom-up models because you need to detect a lot of people in the scene? Just saying, RTMPose is faster and more accurate, and supports HALPE26: https://github.com/open-mmlab/mmpose/tree/main/projects/rtmpose#26-keypoints
![image](https://github.com/user-attachments/assets/343ec0fb-f05a-4029-81b9-ef6cfbf44165)

And RTMLib is a lightweight package that is very easy to install, and I contributed to have it support body+feet estimation. It may be worth trying! Don't hesitate to tell be how it goes :)
https://github.com/Tau-J/rtmlib
